data/templates/frr/fabricd.frr.j2

```diff
@@ -0,0 +1,66 @@
+!
+{% if interface is vyos_defined %}
+{%     for iface, iface_config in interface.items() %}
+interface {{ iface }}
+ ip router openfabric VyOS
+ ipv6 router openfabric VyOS
```

Comment:
Is there a reason why we shouldn't make IPv4 and IPv6 OpenFabric individually configurable?